src/datasource.ts

```diff
@@ -416,9 +416,12 @@ export default class IrondbDatasource {
               var leaf_name = result[i]['metric_name'];
               if (target.labeltype !== "default") {
                 var metriclabel = target.metriclabel;
-                if(target.labeltype === "name") {
+                if (target.labeltype === "name") {
                   metriclabel = "%n";
                 }
+                else if (target.labeltype === "cardinality") {
+                  metriclabel = "%n|ST[%t-{*}]";
```

Comment:
Use `"%n | %t-{*}"` so that this metric name isn't confused with a canonical form.  If we present it as |ST[...] someone could reasonably expect to find that exact metric in the system, but it is likely missing some tags at this point.